tests/test_chatphoto.py

```diff
@@ -185,6 +175,16 @@ async def test_get_and_download(self, bot, chat_photo):
 
         assert jpg_file.is_file()
 
+    async def test_send_all_args(
```

Comment:
if tests need to be in a specific order her to pass, please leave comments in the respective tests explaining why they need to be before/after certain other tests

Reply:
not a problem after https://github.com/python-telegram-bot/python-telegram-bot/pull/3426/commits/4b55ee567ae31bef0f7a0a200daf50351829c8bf